docs/_data/2019.portland.speakers.yaml

```diff
@@ -302,29 +302,6 @@
     twitter: shannon_crabill
     website: http://shannoncrabill.com
   title: Documenting for Open Source
-- abstract: "<p>“When you’re on an airplane and your life depends on it, comics are\
-    \ going to tell you how to open an emergency exit.”\n       -- Scott McCloud</p>\n\
-    <p>Comics are fun; they’re also an effective way to communicate complex ideas\
-    \ simply. It’s an efficient medium that can delight and inform your most-technical\
-    \ audience while also being comprehensible to non-technical people.</p>\n<p>Comics\
-    \ are also easy to get wrong. Humor can come off offensive or cheesy. Graphical\
-    \ explanations can be impenetrable or patronizing. Creating viral and awesome\
-    \ content takes finesse. </p>\n<p>Speakers Allen Tsai and Syne Mitchell will share\
-    \ with their experience creating comics at Google (ex: https://g.co/dev/gkecomic\
-    \ ). They’ll explain their process, the science behind how comics help explain\
-    \ concepts, as well as show you the results of an A/B test translating one developer\
-    \ topic into comic format.</p>"
-  slug: bam-the-power-of-comics-to-explain-technology-syne-mitchell-allen-tsai
-  speakers:
-  - name: 'Syne Mitchell'
-    slug: syne-mitchell
-    twitter: synemitchell
-    website: https://www.synemitchell.com/   http://www.linkedin.com/in/allen-tsai-ux-lead
-  - name: ' Allen Tsai'
-    slug: allen-tsai
-    twitter: synemitchell
-    website: https://www.synemitchell.com/   http://www.linkedin.com/in/allen-tsai-ux-lead
-  title: Bam! The Power of Comics to Explain Technology
```

Comment:
Should we keep this on the listing, and put a `CANCELLED` in front of it? I'm torn as to what is best.

Reply:
That would be the advantage up generating the link in old news with templates. Unless you *really* value the archive factor of old content.

Reply:
The only breaking broken link (:-p) is the schedule, because I've not updated that.

Reply:
We already sent out an email with the old content that we can't change.

Reply:
Yeah, but I don't think that is a huge concern. Let's discuss on slack?

Reply:
Yea, let's just go ahead and kill it then 👍